server/validators/appointmentValidator.test.ts

```diff
@@ -200,8 +265,10 @@ describe('Validation middleware - VLPM appointment', () => {
+      comments: 'a'.repeat(3601),
```

Comment:
Eek! An access token??

Reply:
No this was some ones attempt at putting in comments over the max limit.  I saw it and thought it could be refactored/better.